curriculum/getChallenges.js

```diff
@@ -344,22 +344,23 @@ ${getFullPath('english')}
 // gets the challenge ready for sourcing into Gatsby
 function prepareChallenge(challenge) {
   if (challenge.challengeFiles) {
-    challenge.challengeFiles = challenge.challengeFiles.reduce(
-      (challengeFiles, challengeFile) => {
-        return [
-          ...challengeFiles,
-          {
-            ...createPoly(challengeFile),
-            seed: challengeFile.contents.slice(0)
-          }
-        ];
-      },
-      []
-    );
+    challenge.challengeFiles = filesToPolys(challenge.challengeFiles);
   }
   return challenge;
 }
 
+function filesToPolys(files) {
```

Comment:
I think it's pretty clear from the context that it's challengeFiles.  I can clean it up slightly, so that we don't have the word 'challenge' appear five times on the same line.

> Also, could you explain why this needs to be done twice - once in perpareChallenge, and again in test-challenges.js?

Sure, because solutions are not part of the challenge object, so they don't get `prepareChallenge`d.